playwright/models/VersionHistoryModel.ts

```diff
@@ -0,0 +1,138 @@
+import { expect, Page } from '@playwright/test'
+import { Exam } from 'web/src/types'
+import { ContentType } from '../helpers'
+import { CertificateContentModel } from './CertificateContentModel'
+import { InstructionContentModel } from './InstructionContentModel'
+import { AssignmentContentModel } from './AssignmentContentModel'
+
+export class VersionHistoryModel {
+  constructor(
+    readonly page: Page,
+    readonly exam: Exam,
+    readonly contentType: keyof typeof ContentType,
+    readonly nextVersionButton = page.getByTestId('next-version'),
+    readonly previousVersionButton = page.getByTestId('previous-version'),
+    readonly versionHistoryButton = page.getByTestId('version-history-btn'),
+    readonly versionHistoryModal = page.getByTestId('version-history-modal'),
+    readonly pickVersionButton = page.getByTestId('pick-version')
+  ) {}
+
+  async gotoContentVersionPage(id: number, version: number) {
+    await this.page.goto(`/${this.exam.toLowerCase()}/${this.contentType}/${id}/${version}`)
+  }
+
+  versionHistoryItemSelector(version: number) {
```

Comment:
Nää funkkarit palauttaa Locator:n, ni ehkä nimet vois olla tyylii
```suggestion
  versionHistoryItemLocator(version: number) {
```